REFERENCE.md

```diff
@@ -886,25 +886,28 @@ Default value: `$redis::params::minimum_version`
 
 ### `redis::administration`
 
-Allows various adminstrative settings for Redis
+Allows various administrative settings for Redis
```

Comment:
You should fix the typo in the file itself and then regenerate `REFERENCE.md` using `bundle exec rake strings:generate:reference`.

Reply:
Okay, moved the comment regarding THP to the class itself. Generated reference file but is this many changes expected @ekohl ?